### DISCUSSION TITLE
[V3] How to pass params in view from a dispatched event

### DISCUSSION THREAD

**Discussion by dnextreme88** [livewire/livewire#8711] [1. Help]
I have this in my Livewire controller. This one works but I want to dispatch an event that will only show to a specific element when the request is finished. So far, I am dispatching an event that shows up to the top of the page but I only want to show this inside the element.

![image](https://github.com/user-attachments/assets/460307b9-ba67-4cc0-a6f7-29dcfebd149a)

How do I pass the parameters from a dispatched event to a Livewire component view? I currently have this in my code. I couldn't understand how to pass the params in the Livewire docs either as they're too unclear.

![image](https://github.com/user-attachments/assets/815ef21b-6de5-40e4-8e24-7b6dcd55258e)

**Comment by Alejandrovsdev** [comment]
if u want to show a success message, try 
```php
$this->session()->flash('trainee_id', $document_trainee_id_last_request);
```
then in your view something like:

```blade
        @if (session('trainee_id'))
            your code
        @endif
```
